sourmash/sbt_storage.py

```diff
@@ -142,14 +142,14 @@ def _save_to_zf(self, zf, path, content):
         # so better to have an auxiliary method
         try:
             info = zf.getinfo(path)
+            entry_content = zf.read(info)
 
-            with zf.open(info, mode='r') as entry:
-                if entry.read() == content:
```

Comment:
I removed this because `entry.read()` was returning empty content, and it's more clear to run `zf.read(info)` anyway.